### ISSUE TITLE
Heat Pump Integration

### ISSUE THREAD

**Issue by avinashresearch1** [avinashresearch1/CompHENS.jl#13]
Targetting with heat pumps involved.

**Comment by abcdvvvv** [comment]
Hello avinashresearch1! Have you completed the heat pump integration yet?  If completed, how do I use it?

**Comment by avinashresearch1** [comment]
No, I have not had the bandwidth to get this done. I would be happy to help review and accept PRs though.
